common/rpc-directory/directory.mjs

```diff
@@ -294,6 +294,10 @@ class Directory {
     // TODO: Set them to a spawn point specified by map.json.
     // {MapCoord} mapCoord - Character's map coordinate.
     setDefaultValue(playerData, 'mapCoord', new MapCoord('world1', 10, 10));
+    // {Number} x - Character's x coordinate.
+    setDefaultValue(playerData, 'x', -1);
+    // {Number} y - Characetr's y coordinate.
+    setDefaultValue(playerData, 'y', -1);
```

Comment:
No, character's coordinate is stored in playerData.mapCoord
You should not add x and y like this.